doc/docs/master/concepts/pipeline-concepts/datum/glob-pattern.md

```diff
@@ -4,29 +4,21 @@ Defining how your data is spread among workers is one of
 the most important aspects of distributed computation.
 
 Pachyderm uses **glob patterns** to provide flexibility to
-define data distribution.
-
-TODO: I would instead focus on the fact that PFS inputs are essentially like this (although I wouldn't word it as "glob pattern is applied to the root", rather glob pattern applied to all of the directories / files with the matching ones being the datums produced for the PFS input). Also, datums are actually determined by how the inputs are composed. If you just have a PFS input, then you basically get this. If you have a cross of multiple PFS inputs, then you generally get more datums than input files / directories. 
-
-You can think of each input repository as a filesystem where
-the glob pattern is applied to the root. 
-The files and directories that match the
-glob pattern constitute the [datums](https://docs.pachyderm.com/latest/concepts/pipeline-concepts/datum/)
-that will be processed by the worker(s) that run your pipeline code.
-
-!!! Important
-        You must **configure a glob pattern for each PFS input** of a [pipeline specification](). 
+define data distribution. 
 
 !!! Note
      The Pachyderm's concept of glob patterns is similar to the Unix glob patterns.
```

Comment:
I would change `The Pachyderm's` to just `Pachyderm's`. Also `the Unix` to `Unix`.